### PR TITLE
Update clear cache

### PR DESCRIPTION
**Pull request by MaxAstin**
- Deprecated Dao/Entities removed
- Clear cache updated

### CODE REVIEW

File: data/src/main/java/com/bunbeauty/data/repository/OrderRepository.kt [hunk 1]
```diff
@@ -142,6 +142,11 @@ class OrderRepository @Inject constructor(
         }
     }
 
+    override fun clearCache() {
+        cachedOrderList = emptyList()
```
Comment (xidealo) [inline data/src/main/java/com/bunbeauty/data/repository/OrderRepository.kt:146]: make nullable cache

File: data/src/main/java/com/bunbeauty/data/repository/MenuProductRepository.kt [hunk 2]
```diff
@@ -101,7 +99,7 @@ class MenuProductRepository @Inject constructor(
         }
     }
 
-    override suspend fun clearCache() {
-        menuProductDao.deleteAll()
+    override fun clearCache() {
+        menuProductCache = emptyList()
```
Comment (xidealo) [inline data/src/main/java/com/bunbeauty/data/repository/MenuProductRepository.kt:103]: make nullable cache

File: data/src/main/java/com/bunbeauty/data/repository/OrderRepository.kt (hunk 1 above)
Comment (MaxAstin) [reply data/src/main/java/com/bunbeauty/data/repository/OrderRepository.kt]: fixed

File: data/src/main/java/com/bunbeauty/data/repository/MenuProductRepository.kt (hunk 2 above)
Comment (MaxAstin) [reply data/src/main/java/com/bunbeauty/data/repository/MenuProductRepository.kt]: fixed

**Review by xidealo — APPROVED** [review]

**Pull request merged**